src/screen/ParathreadsScreen.tsx

```diff
@@ -0,0 +1,146 @@
+import {EmptyView} from 'presentational/EmptyView';
+
+type ParathreadData = {
+  name?: string;
+  homepage?: string;
+};
+
+export function ParathreadsScreen() {
+  const {data: leasePeriod} = useParachainsLeasePeriod();
+  const {data: parathreads, isLoading} = useParathreads();
+
+  const toHomepage = (url?: string) => {
+    if (url && Linking.canOpenURL(url)) {
+      Linking.openURL(url);
+    }
+  };
+
+  return (
+    <SafeView edges={noTopEdges}>
+      {isLoading ? (
+        <View style={globalStyles.centeredContainer}>
+          <Spinner />
+        </View>
+      ) : (
+        leasePeriod && (
+          <FlatList
+            style={globalStyles.flex}
+            contentContainerStyle={styles.content}
+            keyExtractor={([id, _]) => id.toString()}
+            data={parathreads}
+            renderItem={({item}) => <ParathreadItem id={item[0]} leases={item[1]} toHomepage={toHomepage} />}
+            ItemSeparatorComponent={Divider}
+            ListEmptyComponent={EmptyView}
+          />
+        )
+      )}
+    </SafeView>
+  );
+}
+
+type ParathreadItemProps = {
+  id: ParaId;
+  leases: LeaseInfo[];
+  toHomepage: (url?: string) => void;
+};
+
+function ParathreadItem({id, toHomepage}: ParathreadItemProps) {
+  const {data: parathreadInfo} = useParathreadInfo(id);
+  const parathreadData = useParathreadData(id);
+
+  return (
+    <Card style={styles.card} onPress={() => toHomepage(parathreadData.homepage)}>
+      <View style={[styles.row, {justifyContent: 'space-between'}]}>
+        <View style={styles.row}>
+          {parathreadInfo?.manager && (
+            <View style={styles.chainImage}>
+              <Identicon value={parathreadInfo.manager} size={30} />
+            </View>
+          )}
+          {parathreadData.name && (
+            <Text category="s1" numberOfLines={1} status="info">
+              {parathreadData.name}
+            </Text>
+          )}
+        </View>
+        <Text category={'c1'} numberOfLines={1} ellipsizeMode="middle" appearance="hint">
+          {formatNumber(id)}
+        </Text>
+      </View>
+
+      {parathreadInfo?.manager && (
+        <View style={[styles.row, styles.marginTop]}>
+          <Text category="c1" appearance="hint">
+            manager:{' '}
+          </Text>
+          <Account id={parathreadInfo.manager.toString()}>
+            {(identity) => (
+              <View style={styles.accountsRow}>
+                {identity?.accountId && <Identicon value={identity.accountId} size={20} />}
+                <Padder scale={0.3} />
+                {identity?.display && (
+                  <View style={styles.row}>
+                    <Text numberOfLines={1} category={'c1'}>
+                      {identity.display}
+                    </Text>
+                  </View>
+                )}
+              </View>
+            )}
+          </Account>
+        </View>
+      )}
+      {/* @TODO: Add leases information */}
```

Comment:
Are you planning to do this in another PR?

Reply:
Right now there are no lease information for the parathreads so I don't have an example of how the data is represented in the UI. If they are the same as parachains I guess it can be done but I decided to skip it for now until we have some actual lease data and final UI design.

So yes, maybe later with another PR, maybe I can create a ticket for it? 🤔

Reply:
Yeah let's create an issue so we don't forget 👍🏼